supplementary/alica_ros_proxy/src/logger/AlicaRosLogger.cpp

```diff
@@ -0,0 +1,56 @@
+#include <logger/AlicaRosLogger.h>
+
+#include <iostream>
+#include <ros/callback_queue.h>
+#include <ros/console.h>
+#include <ros/ros.h>
+#include <sstream>
+
+using namespace alica;
+
+namespace alicaRosLogger
+{
+AlicaRosLogger::AlicaRosLogger(Verbosity verbosity, std::string localAgentName, int64_t localAgentId)
+        : _localAgentName(localAgentName)
+        , _localAgentId(localAgentId)
+{
+    ros::console::Level level;
```

Comment:
`level = INFO`
nit: define static const unordered_map & initialize in the initializer list itself